search/api.py

```diff
@@ -29,7 +29,7 @@ def execute_search(search_obj):
     return search_obj.execute()
 
 
-def create_program_limit_query(user):
+def create_program_limit_query(user, omit_email_optin_false=False):
```

Comment:
Please rename to `omit_email_optin`, and please add information on the arguments and return value to the docstring